config/crd/bases/crdb.cockroachlabs.com_crdbclusters.yaml

```diff
@@ -1089,6 +1089,9 @@ spec:
           status:
             description: CrdbClusterStatus defines the observed state of Cluster
             properties:
+              activeActor:
+                description: ActiveActor keeps track of which Actor is currently running
```

Comment:
I think we want this in an annotation or status rather than a field for the crd.  Unless a user needs to modify this field.

Reply:
Doh! Correct